modules/tide_site_restriction/tide_site_restriction.module

```diff
@@ -394,3 +398,39 @@ function tide_site_restriction_views_data_alter(array &$data) {
 function tide_site_restriction_views_plugins_field_alter(array &$plugins) {
   $plugins['node_bulk_form']['class'] = 'Drupal\\tide_site_restriction\\Plugin\\views\\field\\TideNodeBulkForm';
 }
+
+/**
+ * Implements hook_field_widget_single_element_form_alter().
+ */
+function tide_site_restriction_field_widget_single_element_form_alter(&$element, FormStateInterface $form_state, $context) {
```

Comment:
this function is to show all the sites and if there is any child site under it during editing other users.